crates/sui-open-rpc/src/examples.rs

```diff
@@ -1040,7 +1128,7 @@ impl RpcExampleProvider {
 
         Examples::new("suix_getDynamicFields",             
         vec![ExamplePairing::new(
-            "Get all the dynamic fields of a particular object. Return a paginated list of `limit` results per page. The current limit is 50 per page.",
+            "Get dynamic fields for the object the request provides. Return a paginated list of `limit` dynamic field results per page, beginning with the dynamic field immediately after the provided object ID. The default limit is 50.",
             vec![
```

Comment:
Note that the cursor returned is only a pagination id and does not correlate to the last object ID in the returned vector.